server/Customer.java

```diff
@@ -22,30 +26,37 @@ public String accountsToString() {
 		return s;
 	}
 
-	public void addAccount(Account account) {
+	public void addAccount(newbank.server.Account account) {
 		accounts.add(account);
 	}
 
-	public ArrayList<Account> getAccounts() {
+	public ArrayList<newbank.server.Account> getAccounts() {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it